src/datumaro/plugins/data_formats/datumaro/base.py

```diff
@@ -174,6 +178,21 @@ def _parse_item(self, item_desc: Dict) -> Optional[DatasetItem]:
                 if self.media_type == MediaElement:
                     self.media_type = PointCloud
 
+            video_frame_info = item_desc.get("video_frame")
+            if media and video_frame_info:
+                raise MediaTypeError("Dataset cannot contain multiple media types")
+            if video_frame_info:
+                video_path = osp.join(self._video_dir, video_frame_info.get("video_path"))
+                if video_path not in self._videos:
+                    self._videos[video_path] = Video(video_path)
+                video = self._videos[video_path]
+
+                frame_index = video_frame_info.get("frame_index")
+
+                media = VideoFrame(video, frame_index)
+                if self.media_type == MediaElement:
+                    self.media_type = VideoFrame
+
```

Comment:
Can you add some more tests to increase test coverage? (not necessarily limited to this line)

Reply:
I added a unit test to archive 80>% coverage